app.py

```diff
@@ -1,13 +1,91 @@
-from flask import Flask
-import requests
+from flask import Flask, render_template, redirect, request
+from flask_login import LoginManager, login_user, logout_user, current_user, login_required
+from flask_mongoengine import MongoEngine
+from flask_mongoengine.wtf import model_form
+from wtforms import PasswordField
+
 
 app = Flask(__name__)
+app.config["DEBUG"] = True
+app.config['MONGODB_SETTINGS'] = { 'db' : 'users' }
+app.config['SECRET_KEY'] = 'JJAM'
+app.config['WTF_CSRF_ENABLED'] = True
```

Comment:
This is fine for now, but as we move to production we'll want to redo the config stuff.